.travis.yml

```diff
@@ -20,7 +20,7 @@ install:
 
 # command to run tests
 script:
-  - python backend/manage.py test
+  - "cd backend && python manage.py test --keepdb && cd .."
```

Comment:
Using `python backend/manage.py test` does not detect any tests.
Maybe `BASE_DIR` problem?